### ISSUE TITLE
Proposal for translation tool

### ISSUE THREAD

**Issue by ilyaspiridonov** [programminghistorian/jekyll#1571]
Please consider using [GitLocalize](https://gitlocalize.com) to manage translations. 
It's free, supports .md and syncs with GitHub directly.

**Comment by svmelton** [comment]
Thanks—I'll let our Spanish and French Managing Editors take a look.

**Comment by mdlincoln** [comment]
jumping in to note that this would not handle the particular way that we do language snippets, so it couldn't handle all our translation needs. 

However, I would be interested to get feedback on how it works for translating a markdown file in ph-submissions. If the @programminghistorian/french-team or @programminghistorian/spanish-team have a specific lesson they'd like to experiment with, let the @programminghistorian/technical-team team know and we could try it out as an option.

**Comment by arojascastro** [comment]
Interesting! I think we should experiment with it. I am planning to translate one lesson in the first semester of 2020 and I could use it. But feel free to experiment with this tool as well in parallel!

**Comment by jenniferisasi** [comment]
I watched their video and was wondering that @mdlincoln, it didn't look to me like the system we use in the website/jekyll folder. 

For what you [I see @arojascastro too] suggest, I see it useful to do a review on EN | ES versions (in my case) but I would personally not use it to do the translation of a lesson itself - that would mean there is a ticket open while in-progress but as of now we only accept completed translations. Makes sense?

**Comment by mdlincoln** [comment]
Actually, because this involves a github integration directly onto our repositories, we really should pick just 1 lesson to start with.

**Comment by mdlincoln** [comment]
But, as @jenniferisasi notes, only if you think it'd actually be usefuul to try out. There's no reason we have to use it.

**Comment by arojascastro** [comment]
I did not watch any video or read the documentation, so I do not know if it *is* really useful! Sorry!

**Comment by mdlincoln** [comment]
Well hey, maybe I'll try setting up a little demo in the ph-submissions repo on some sample lessons (I'll make new temporary directories so it won't interfere with actual ongoing lessons) and you call can take a look before we try using it for a "real" lesson.

**Comment by mdlincoln** [comment]
Hi @programminghistorian/french-team and @programminghistorian/spanish-team I'm trying to assess the number of hours I spend on PH stuff, and this is probably going to be beyond my capacity to work on right now. If there is strong interest from your teams, we can see if another @programminghistorian/technical-team member is able to take point - otherwise I may put this issue into the "new features backlog". Please respond on this ticket if you feel very strongly about trying this out.

**Comment by arojascastro** [comment]
Thanks @mdlincoln - I would say is not a priority or requirement, but desirable... :)

**Comment by JoshuaGOB** [comment]
I agree with @arojascastro. This would be a nice feature but probably more for some of the in-house translations than for submitted ones. 

I’d love to dig in at a later point since I do use TM and MT software for work and providing those glossaries might help on longer lessons.

**Comment by mdlincoln** [comment]
OK, let me raise this with the rest of the tech team and see if someone else might be able to do some experimentation in the next month or two. @JoshuaGOB you raise a good point about this being useful for e.g. keeping the editor guidelines in sync, rather than for brand new translations. Perhaps we can look into that as the first example for experimentation.

**Comment by spapastamkou** [comment]
I'll make the test for this once I am on my next translation; possible schedule on April 2020.

**Comment by mdlincoln** [comment]
Hello @spapastamkou - I wanted to check on this; I know our commitments have changed radically around the globe since February. Are you still going to be able to try this out in the next month? Or should we close this due to lack of time? I'm happy with either decision, I just want to make sure that we make a decision and then adjust the ticket.

**Comment by spapastamkou** [comment]
Hi @mdlincoln, yes, I kind of launched this on April 19: the lesson I will work on is "Downloading Multiple Records Using Query Strings". I uploaded the lesson on our repo and tried to make the connexion with GitLocalize. I did not completely figure out this, and had not much time then, but it is in my pipeline. Once launched, I will open a proposal ticket in the ph-submissions repo, and either close this ticket then or wait until done and report here?

**Comment by mdlincoln** [comment]
@spapastamkou Great, thank for letting us know you're trying it out on that lesson. I'd report back on what you find out about the tool on this ticket, and then we can figure out if it's really something that the project team would like to use more widely.

**Comment by spapastamkou** [comment]
ok, let's do this.

**Comment by mariajoafana** [comment]
Looking forward to seeing the results of this experiment

**Comment by acrymble** [comment]
With @mdlincoln stepping down and our team entering a phase of reduced new technical features, is this still realistic as a widespread solution?

**Comment by DanielAlvesLABDH** [comment]
I gave a look at the website and the video of this tool. It has potential to be helpful but probably it will take some effort to integrate with our current processes. I could try to ask someone from the PT team to give it a try, but with no promises since we are all with too much work at the time. Please let me know what you think.

**Comment by mdlincoln** [comment]
Thanks @DanielAlvesLABDH but you have enough work to do now. It's clear no one has bandwidth for this now. I'm moving it to our backlog.

**Comment by mdlincoln** [comment]
I spoke too soon - @spapastamkou has still been working on evaluating this tool, and hopes to give a more detailed report out to the full team after her sabbatical. I'm going to re-open to mark that this work is actively ongoing.

**Comment by mariajoafana** [comment]
@spapastamkou Shall we close this ticket and re-open when this moves along?

**Comment by acrymble** [comment]
I'm closing this because we haven't had any movement on it for some time and keeping a ticket open doesn't affect anyone's ability to look into it further. It doesn't seem to be an active task of great importance for us at the moment.
